src/reducers/index.js

```diff
@@ -1,10 +1,12 @@
 // Copyright 2018, 2019 Stanford University see LICENSE for license
+/* eslint complexity: ["warn", 13] */
 
 import { combineReducers } from 'redux'
 import shortid from 'shortid'
 import authenticate from './authenticate'
 import {
-  removeAllContent, removeMyItem, setMyItems, setMySelections, setBaseURL, setMyItemsLang, validate,
+  removeAllContent, removeMyItem, setMyItems, setMySelections, setBaseURL, setMyItemsLang,
+  validate, showGroupChooser,
```

Comment:
same question as above -- if we're importing this many pieces, would it be better to import the whole thing?

Reply:
I like being explicit here. I think it makes it easier to tell what we're actually using.

Reply:
i vote for leaving like this, but am happy to go with consensus